packages/dev/codemods/src/s1-to-s2/src/codemods/changes.ts

```diff
@@ -426,19 +443,6 @@ export const changes: ChangesJSON = {
       }
     ]
   },
-  Flex: {
```

Comment:
realized these transforms aren't needed since this is already handled in `styleProps.ts`